network-api/networkapi/buyersguide/templates/bg_base.html

```diff
@@ -67,7 +67,7 @@
 
       <footer class="mofo-footer">
         <div class="container">
-          <ul class="list-unstyled footer-links row justify-content-center">
+          <ul class="list-unstyled footer-links row justify-content-center mb-0">
```

Comment:
`mb-0` (margin bottom 0): a bootstrap class.

Reply:
Sounds good, I'll look through the project as well tomorrow.